trojsten/top30/templates/trojsten/top30/top30.html

```diff
@@ -0,0 +1,39 @@
+{% extends "trojsten/layout/main.html" %}
+{% load i18n %}
+
+{% block title %}TOP 30{% endblock %}
+
+{% block page_header %}
+
+<h1> TOP 30 </h1>
+
+{% endblock %}
+
+{% block page_content %}
+
+{% for heading, leaderboard in leaderboards %}
+    {% if leaderboard %}
+        <h2> {{ heading }} </h2>
+        
+        <table>

```

Comment:
Nechceme tu nejake bootstrap classes aby to pekne vyzeralo?

Reply:
Nemám veľký cit pre pekný dizajn a nevyznám sa v CSS/Bootstrape, ale kľudne navrhni nejakú classu a dám ju tam.

Reply:
Tot ako sa robia tabulky v bootsrtape: https://getbootstrap.com/docs/3.3/css/#tables

Odporucam hover rows: https://getbootstrap.com/docs/3.3/css/#tables-hover-rows
TO je konzistentne aj s vysledkovkou: https://www.ksp.sk/vysledky/